packages/nhost_dart/pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
   nhost_graphql_adapter: ^4.0.5
 dev_dependencies:
   fake_async: ^1.3.1
-  graphql: ^5.1.3
```

Comment:
please, let's not upgrade to beta versions unless there is a very good reason for it